abort/abort.fork.test.ts

```diff
@@ -0,0 +1,205 @@
+import { createDomain } from 'effector';
+import { fork, allSettled, scopeBind } from 'effector/fork';
+import { abort, AbortedError } from './index';
+
+test('Abort: aborts all effects calls with the same key', async () => {
+  const app = createDomain();
+  const signal = app.createEvent<string>();
+  const fx = abort({
+    domain: app,
+    signal,
+    getKey: () => 'shared',
+    async handler(_: void, { onAbort }) {
+      await new Promise((r, rj) => {
+        const id = setTimeout(r, 10);
```

Comment:
Such small timeouts could lead to flaky tests. In some cases, it's better to replace them with mocks https://jestjs.io/docs/timer-mocks

Reply:
I'm not sure, if its applaible here 🤔
Especially in fork tests, where `allSettled` is available, might make sense for out-of-scope tests though

Reply:
👌